sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -386,9 +386,13 @@ var logsStaticTriggerConditionSchema = map[string]*schema.Schema{
 			"threshold_type": &thresholdTypeSchema,
 		}),
 		"resolution": nested(false, schemaMap{
-			"threshold":         &thresholdSchema,
-			"threshold_type":    &thresholdTypeSchema,
-			"resolution_window": &timeRangeSchema,
+			"threshold":      &thresholdSchema,
+			"threshold_type": &thresholdTypeSchema,
+			"resolution_window": {
```

Comment:
DRY - We can define a `var` for it and re-use at all these places.

Reply:
For some reason it didn't reference the variable timeRangeSchema so went this approach. Will explore more

Reply:
Done.